unit_tests/utilities/test_zaza_utilities_generic.py

```diff
@@ -208,6 +208,13 @@ def test_reboot(self):
             ['juju', 'ssh', _unit,
              'sudo', 'reboot', '&&', 'exit'])
 
+    def test_juju_reboot(self):
+        _unit = "app/2"
+        generic_utils.juju_reboot(_unit)
+        self.subprocess.check_call.assert_called_once_with(
+            ['juju', 'ssh', _unit,
+             f'sudo juju-run -u {_unit} "juju-reboot --now"'])
```

Comment:
We tend not to use f-strings in zaza purely due to the python3.5 support.  We need to formally make a break on it, but zaza still tentatively supports py35!

Reply:
Addressed in https://github.com/openstack-charmers/zaza/pull/575/commits/7417002766b3ba808bae185755183fb64b83a31b

Reply:
This test code still uses the f-string?  I see the code-under-test does use `.format()`.